src/models/room.ts

```diff
@@ -2436,17 +2391,17 @@ export class Room extends TypedEventEmitter<EmittedEvents, RoomEventHandlerMap>
                 }
 
                 if (shouldLiveInThread) {
```

Comment:
maybe the check for threadId should be in the if condition instead? Feels a bit code-smelly to be setting threadId to an empty string

Reply:
Yes that's what I thought, but doing that would also prevent it from being added to the correct partition and would therefore not end up in any timeline. Which sounds unfortunate 🤔